### ISSUE TITLE
Missing type stub for _add_value_alias_ method in Python 3.13 enum

### ISSUE THREAD

**Issue by tjdcs** [microsoft/pyright#10707]
## Bug Report

**Describe the bug**
Pyright reports an attribute access error for the  method that was added to Python's enum module in version 3.13.

**Error Message**
```
Cannot access attribute "_add_value_alias_" for class "EOTFType*"
Attribute "_add_value_alias_" is unknown (reportAttributeAccessIssue)
```

**Code Sample**
```python
from enum import Enum
from typing import Any

class EOTFType(str, Enum):
    RESERVED = ("RESERVED", 0)
    SDR = ("SDR", 1)
    PQ = ("PQ", 2)
    HLG = ("HLG", 3)

    def __new__(cls, value: str, *args: Any):
        self = str.__new__(cls, value)
        self._value_ = value
        for a in args:
            self._add_value_alias_(a)  # Pyright error here
        return self
```

**Expected behavior**
Pyright should recognize `_add_value_alias_` as a valid method on enum classes in Python 3.13.

**Environment**
- Pyright version: 1.1.403
- Python version: 3.13
- Platform: macOS

**Additional context**
According to the [Python 3.13 enum documentation](https://docs.python.org/3/library/enum.html), `_add_value_alias_` was added in version 3.13 as a method of `EnumType` to "add a new value as an alias to an existing member".

The method works correctly at runtime but Pyright's type stubs appear to be missing this new method. This affects code that uses this legitimate Python 3.13 enum feature.

**Workaround**
Currently using `# type: ignore` comment to suppress the error, but proper type stub support would be preferred.

**Comment by tjdcs** [comment]
* This issue was written by Claude Sonnet 4 (model ID: claude-sonnet-4-20250514) (claude code) during development and reviewed by me before reporting. I did additional searches on the same issue and couldn't find it reported.

**Comment by erictraut** [comment]
In general, pyright doesn't have any hard-coded knowledge of stdlib types. This type information comes from a community-maintained set of type stubs in the typeshed project. For example, the fact that the `enum` class defines an attribute named `_value_` is specified [here](https://github.com/python/typeshed/blob/9a0eaf8df5b37a4351c61af6e8e510f57e212131/stdlib/enum.pyi#L198) in the `enum.pyi` stub.

You can see that there is no definition for `_add_value_alias_` in this class, which is why pyright reports an error here. If you think that `_add_value_alias_` should be defined here, please consider filing a bug report in the typeshed project — or better yet, a PR. All of the type checkers and other static analysis tools in the Python ecosystem leverage typeshed, so improvements like this benefit everyone.

I regularly pull the latest updates to typeshed stubs and ship them with pyright, so any improvements in typeshed will automatically appear in future releases of pyright.

I'm going to close this issue because this isn't a bug in pyright.

**Comment by tjdcs** [comment]
Thanks! My apologies for reporting this in the wrong place!
